packages/gamut/src/ContentContainer/index.tsx

```diff
@@ -0,0 +1,34 @@
+import cx from 'classnames';
+import React, { HTMLAttributes, ReactHTML } from 'react';
+
+import s from './styles.module.scss';
+
+export type ContentContainerProps = HTMLAttributes<HTMLElement> & {
+  el?: keyof ReactHTML;
+  wide?: boolean;
+};
+
+const ContentContainer: React.FC<ContentContainerProps> = ({
```

Comment:
```suggestion
export const ContentContainer: React.FC<ContentContainerProps> = ({
```

This'll fix your build errors.

Reply:
@JoshuaKGoldberg, any insight on why this snapshot test could be failing even after this fix?
https://circleci.com/gh/Codecademy/client-modules/21050#tests/containers/3

Reply:
Aha...`packages/gamut/__tests__/__snapshots__/gamut-test.ts.snap`?

Reply:
Had to run `yarn test -u` to update the snapshot in that file.